app/main.py

```diff
@@ -1 +1,6 @@
-# write your code here
+def copy_file(command: str) -> None:
+    command = command.split(" ")
```

Comment:
space is default in split
```suggestion
    command = command.split()
```